mods/ra/maps/allies-05a/weapons.yaml

```diff
@@ -2,19 +2,14 @@ MissionColt:
 	ReloadDelay: 6
 	Range: 7c0
 	Report: gun5.aud
-	Projectile: Bullet
-		High: true
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
 		Spread: 42
 		Damage: 50
 		Versus:
-			None: 0
 			Wood: 10
-			Light: 0
-			Heavy: 0
-			Concrete: 0
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
+		AffectsParent: true
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 		InvalidImpactTypes: Water
```

Comment:
Westwoods' engine wasn't as flexible as ours, and they were a lazy bunch. Also, 'piffs' is wrong either way (should have been 'piff'), but I agree with @obrakmann. 👍 after that.